spec/Section 6 -- Execution.md

```diff
@@ -734,9 +734,22 @@ the {objectValue}.
 
 ResolveAbstractType(abstractType, objectValue):
 
-- Return the result of calling the internal method provided by the type system
-  for determining the Object type of {abstractType} given the value
-  {objectValue}.
+- Let {resolvedType} be the result of calling the internal method provided by
+  the type system for determining the Object type of {abstractType} given the
+  value {objectValue}.
+- Assert {resolvedType} is an Object type.
+- If {IsSubType(abstractType, resolvedType)}:
+  - Return {resolvedType}.
+- Otherwise, raise a field error.
+
+IsSubType(superType, maybeSubType):
+
+- If {abstractType} is an Interface type.
+  - If {resolvedType} implements {abstractType}, return {true}.
+  - Otherwise, return {false}.
+- If {abstractType} is a Union type.
+  - If {resolvedType} is a member type of {abstractType}, return {true}.
+  - Otherwise, return {false}.
```

Comment:
`superType` isn't used, and I think `abstractType` is clearer and more consistent with the calling method. `maybeSubType` is problematic for me, the term "maybe" often implies nullability, and "sub" doesn't feel right as mentioned above. How about `otherType`?

```suggestion
IsAbstractMemberType(abstractType, otherType):

- Assert: {abstractType} is an Interface or Union type.
- If {abstractType} is an Interface type.
  - If {otherType} implements {abstractType}, return {true}.
  - Otherwise, return {false}.
- If {abstractType} is a Union type.
  - If {otherType} is a member type of {abstractType}, return {true}.
  - Otherwise, return {false}.
```

Reply:
corrected as [c15550f](https://github.com/graphql/graphql-spec/pull/973/commits/c15550f287539cf5b7009ca389e8b7a46495ede1)

in terms of the language of "sub-type", it is used elsewhere, e.g. within [IsValidImplementation](https://spec.graphql.org/draft/#IsValidImplementation()):

> field must return a type which is equal to or a sub-type of (covariant) the return type of implementedField field’s return type

Reply:
In fact, I think if #960 is adopted, {objectType} could actually be an Interface type, and then a portion of [IsValidImplementationFieldType](https://spec.graphql.org/draft/#IsValidImplementationFieldType()) could be replaced by IsSubType => as the difference there is that object types or interface types are currently inspected as to whether they are subtypes, and we are just inspecting object types. Maybe a different PR should abstract that out anyway, and even though the algorithm here only inspects object types, we could copy that behavior?

Reply:
See #977